.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": [
+            "error",
```

Comment:
This file was auto-generated by eslint --init. Near as I can tell, "error" is the expected value here: https://github.com/eslint/eslint/blob/4133a38447b9cdfef25efd91083bdbcb5ece4ccb/tests/lib/config/config-initializer.js#L117. There is an open bug about this error. It suggests this format is new, and there is some uncertainty about whose bug it is. But I think not mine. :) https://github.com/AtomLinter/linter-eslint/issues/471#issuecomment-193878811